loops/swift/code.swift

```diff
@@ -0,0 +1,21 @@
+@main
+struct Main {
+    static func main() {
+        let u = Int(CommandLine.arguments[1])!
+        let r = Int.random(in: 0..<10000)
+
+        withUnsafeTemporaryAllocation(of: Int.self, capacity: 10000) { a in
+            a.initialize(repeating: 0)
+
+            for i in 0..<10000 {
```

Comment:
Since all the values are `0` from the `a.initialize(repeating: 0)` line, the first iteration can be skipped entirely

```suggestion
            for i in 1..<10000 {
```